locale/fr/crud.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+return [
+    'add_new' => 'Ajouter',
+    'cancel' => 'Annuler',
+    'save' => 'Sauvegarder',
+    'delete' => 'Supprimer',
+    'send' => 'Envoyer',
+    'detail' => 'Détail',
+    'back' => 'Retour',
```

Comment:
Je ne sais pas si c'est un fix correct mais j'ai remarqué que dans nos stubs, on utilise `crud.back_to_list` plutôt que seulement `crud.back`.
```suggestion
    'back_to_list' => 'Retour',
```